types/bootstrap-select/index.d.ts

```diff
@@ -8,42 +8,90 @@
 /// <reference types="jquery"/>
 
 interface BootstrapSelectOptions {
-    actionsBox?: boolean | undefined;
-    container?: string | boolean | undefined;
-    countSelectedText?: string | Function | undefined;
-    deselectAllText?: string | undefined;
-    dropdownAlignRight?: string | boolean | undefined;
-    dropupAuto?: boolean | undefined;
-    header?: string | undefined;
-    hideDisabled?: boolean | undefined;
-    iconBase?: string | undefined;
-    liveSearch?: boolean | undefined;
-    liveSearchNormalize?: boolean | undefined;
-    liveSearchPlaceholder?: string | undefined;
-    liveSearchStyle?: string | undefined;
-    maxOptions?: number | boolean | undefined;
-    maxOptionsText?: string | Array<any> | Function | undefined;
-    mobile?: boolean | undefined;
-    multipleSeparator?: string | undefined;
-    noneSelectedText?: string | undefined;
-    noneResultsText?: string | undefined;
-    selectAllText?: string | undefined;
-    selectedTextFormat?: string | undefined;
-    selectOnTab?: boolean | undefined;
-    showContent?: boolean | undefined;
-    showIcon?: boolean | undefined;
-    showSubtext?: boolean | undefined;
-    showTick?: boolean | undefined;
-    sanitize?: boolean | undefined;
-    sanitizeFn?: ((unsafeElements: Array<HTMLElement | ChildNode | Node>) => void) | undefined;
-    size?: 'auto' | number | boolean | undefined;
-    style?: string | undefined;
-    styleBase?: string | undefined;
-    tickIcon?: string | undefined;
-    title?: string | undefined;
-    virtualScroll?: boolean | number | undefined;
-    width?: string | boolean | undefined;
-    windowPadding?: number | number[] | undefined;
+    actionsBox: boolean;
```

Comment:
These interfaces have almost the same structure. It would be better to extract properties, which exists in `BootstrapSelectOptions` and `BootstrapSelectDefaults`, into base type/interface.